eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -41,6 +41,8 @@ stages:
       - stage: Release_${{artifact.safeName}}
         displayName: 'Release: ${{artifact.name}}'
         dependsOn: Signing
+        variables: 
+          SetDevVersion: $[stageDependencies.Build.Build.outputs['SetupVersioningProperties.SetDevVersion']]
```

Comment:
Do we also need to do this in the Integration stage?

Reply:
I wouldn't expect this variable to be set before the condition is evaluated. I would expect you to need to directly reference the output variable in the condition.

Reply:
Actually I thought I was putting it on the integration stage :) I'll fix this up, I'm going through the process of merging the eng/common change right now. Then I can come back to this and test it all out.